### DISCUSSION TITLE
How do you run the app on Windows?

### DISCUSSION THREAD

**Discussion by ghost** [Breakthrough/DVR-Scan#177] [Q&A] [answered]
Hello, I am very new to things like this. I installed the latest install of 1.6.1 MSI for Windows. But I don't see an app anywhere to click to open to use it. Does anyone know how to open the app and have it show up on my computer? Thank you!

**Comment by Breakthrough** [comment, marked as answer]
See #178 for some helpful shortcuts to allow drag-and-drop for running DVR-Scan.

DVR-Scan is mostly a command-line program, however that changed in v1.6 with the addition of the region editor.  I can look into adding some helpful batch files or shortcuts to allow you to drag-and-drop files to be scanned.  I also will look into adding a file pop-up to select files when you launch the app.

I'll file an issue to track this work for the next release.  In the meantime I'll look into some workarounds you can use like desktop shortcuts.  Let me get back to you on that.

**Reply by Breakthrough** [reply]
Filed #178 to track this, and uploaded a ZIP there with some shortcuts to let you drag-and-drop videos to run DVR-Scan.  Let me know if this helps out 😄
